qemu/tests/block_copy.py

```diff
@@ -141,13 +141,23 @@ def is_paused(self):
         """
         Return block job paused status.
         """
-        paused = self.get_status().get("paused")
-        offset_p = self.get_status().get("offset")
-        time.sleep(random.uniform(1, 3))
-        offset_l = self.get_status().get("offset")
-        paused &= offset_p == offset_l
+        time.sleep(5)
+        paused, offset_p = self.paused_status()
+        if paused:
+            time.sleep(random.uniform(1, 3))
+            paused_l, offset_l = self.paused_status()
+            paused &= offset_p == offset_l and paused_l
         return paused
 
+    def paused_status(self):
+        """
+        Get key value for pause status.
+        """
+        status = self.get_status()
+        paused = status.get("paused") and not status.get("busy")
```

Comment:
@xutian @luckyh suggested that check "paused" and "busy" twice when get offset, to avoid potential bz that maybe these status changed when get later offset(if it increases), I think more check will be safer, so finally modify like this.